src/main/archetype/all/pom.xml

```diff
@@ -52,7 +52,7 @@
                     <group>${groupId}</group>
                     <packageType>container</packageType>
                     <properties>
-                        <cloudManagerTarget>all</cloudManagerTarget>
+                        <cloudManagerTarget>none</cloudManagerTarget>
```

Comment:
@bpauli I think this line should just be removed -- the result of this change would be that *all* of the generated packages would have a `none` target, which would be an error condition on Cloud Service.

But a different way, since `none` is the only supported value, `<cloudManagerTarget>all</cloudManagerTarget>` is functionally equivalent to *not* specifying a `cloudManagerTarget` property at all.

Reply:
@justinedelson is right and this is clearly documented at https://experienceleague.adobe.com/docs/experience-manager-cloud-service/implementing/developing/aem-project-content-package-structure.html#marking-packages-for-deployment-by-adoube-cloud-manager
> By default, Adobe Cloud Manager harvests all packages produced by the Maven build, however since the container (all) package is the singular deployment artifact that contains all code and content packages, we must ensure only the container (all) package is deployed. To ensure this, other Packages the Maven build generates must be marked with the FileVault Content Package Maven Plug-In configuration of <properties>`<cloudManagerTarget>none</cloudManageTarget></properties>`.